### ISSUE TITLE
Property completion on template always include =

### ISSUE THREAD

**Issue by fgallardograzio** [angular/vscode-ng-language-service#172]
When a property is autocompleted in a template, an `=` sign gets added at the end, no matter what.

For example, this:
```html
<button (cli|></button>
```

Gets autocompleted to this:
```html
<button (click)=|></button>
```

If I go and change the property, this:
```html
<button (dblc|)=></button>
```

Gets autocompleted to this:
```html
<button (dblclick)=|=></button>
```

The closing parenthesis is handled OK, but the equal sign is duplicated. (`|` is the cursor indicator).

As an improvement suggestion, I think quotes should be added. So the first example would get autocompleted to:
```html
<button (click)="|"></button>
```

Thanks!

**Comment by kyliau** [comment]
@ayazhafiz could you please confirm that this is still the case with autocompletion?

**Comment by ayazhafiz** [comment]
So in current versions of the language service it seems that completions like `(click)` do not include `=` or quotes after the `(click)` expression, but parentheses are duplicated; i.e.

`(cl|)` is completed to `((click))`

One way to resolve this is what we discussed "offline"; we should only provide such completions inside parentheses without the parentheses being part of the completion. We could also add an `=` or quotes later on, but this is more complicates.

**Comment by ayazhafiz** [comment]
The issue of parentheses completion was fixed in angular/angular#34445 and a newer discussion about adding quotes to the completion as in the

```html
<button (click)="|"></button>
```

case is hosted at #521; closing to track there.
